examples/react-full/renderer/_default.page.client.tsx

```diff
@@ -10,10 +10,10 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import { PageShell } from './PageShell'
 import { getPageTitle } from './getPageTitle'
-import type { PageContextClient } from './types'
+import type { OnRenderClientAsync } from 'vike/types'
 
 let root: ReactDOM.Root
-async function render(pageContext: PageContextClient) {
+const render: OnRenderClientAsync = async (pageContext): ReturnType<OnRenderClientAsync> => {
```

Comment:
do you think it's fine to add types to non-V1 examples or do you find it misleading? I like it

Reply:
Yes it's fine since the hook types work for the non-V1 design as well (and we actually document them in the current non-V1 docs).


That said, if you feel lazy, you can skip the non-V1 examples (since they'll be removed soon). As you wish.